docs/workflows/common-workflow-steps/maestro-cloud-upload.md

```diff
@@ -0,0 +1,101 @@
+| `$AC_MAESTRO_API_KEY`         | The API key is required for accessing Maestro Cloud services.                                              | Required  |
+| `$AC_MAESTRO_APP_FILE`        | **Android**: Specify the path to an x86 compatible APK file. **iOS**: Provide a zip archive containing an x86 compatible simulator build.   | Required  |
+| `$AC_MAESTRO_WORKSPACE`       | Specifies the directory or file path where Maestro flows are located. By default, it looks for a `.maestro` folder in the project root. Override with a workspace argument if needed.  | Required  |
+| `$AC_MAESTRO_UPLOAD_NAME`     | Specifies the custom name for the upload.                                                               | Optional  |
+| `$AC_MAESTRO_ASYNC`           | Toggle to enable asynchronous mode for running flows.                                                   | Optional  |
+| `$AC_MAESTRO_ENV`             | Pass environment variables to the flows. Separate variables using a new line, or `\n`.                   | Optional  |
+| `$AC_MAESTRO_ANDROID_API_LEVEL` | Set the Android API level for devices to run. The default value is `30`.                              | Optional  |
+| `$AC_MAESTRO_INCLUDE_TAGS`    | Run only flows containing the specified tags (comma-separated).                                         | Optional  |
+| `$AC_MAESTRO_EXCLUDE_TAGS`    | Exclude flows with the specified tags (comma-separated).                                                | Optional  |
+| `$AC_MAESTRO_EXPORT_TEST_REPORT` | Toggle to export the test suite report (JUnit).                                                      | Optional  |
+| `$AC_MAESTRO_EXPORT_OUTPUT`   | Specify the output file for the test file output. The default is `report.xml`.                              | Optional  |
+| `$AC_MAESTRO_MAPPING_FILE`    | **Android**: Include the Proguard mapping file. **iOS**: Include the generated .dSYM file.              | Optional  |
+| `$AC_MAESTRO_BRANCH`          | The branch from which the upload originated.                                                            | Optional  |
+| `$AC_MAESTRO_REPO_NAME`       | The name of the repository (e.g., GitHub repo slug).                                                     | Optional  |
+| `$AC_MAESTRO_REPO_OWNER`      | The owner of the repository (e.g., GitHub organization or user slug).                                    | Optional  |
+| `$AC_MAESTRO_PULL_ID`         | The ID of the pull request from which the upload originated.                                            | Optional  |
+| `$AC_MAESTRO_CLI_VERSION`     | The version of the Maestro CLI is to be downloaded in your CI environment. The default value is the `latest` version.  | Optional  |
+
+### Output Variables
+
+The **Maestro Cloud Upload** step does not produce any output variables. The results are shown in the build log.
```

Comment:
```suggestion
No output variables are produced by the **Maestro Cloud Upload** step; results are displayed in the build log.
```

Reply:
@erenkan I am in favor of keeping the old sentence unchanged for the following reasons:
1. Active Language: The second sentence ("The Maestro Cloud Upload step does not produce any output variables.") uses active language, making it more engaging and direct compared to the passive construction in the first sentence ("No output variables are produced by the Maestro Cloud Upload step").
2. Consistency: The second sentence maintains consistency by using "The Maestro Cloud Upload step" as the subject, which matches the style used in the rest of the document. Consistency in style enhances readability and clarity for the reader.